src/processor/tree/query_tree.rs

```diff
@@ -23,7 +23,7 @@ impl QueryTree {
     pub fn new(db_path: &Path) -> Self {
         assert!(db_path.exists());
 
-        let db = RocksDBWrapper::new(db_path);
+        let db = RocksDBWrapper::new(db_path).unwrap();
```

Comment:
```suggestion
        let db = RocksDBWrapper::new(db_path)?;
```
Let's make this function return `Result<Self>` instead

Reply:
OK